locale/en/messages.json

```diff
@@ -939,6 +940,7 @@
   "your service's requirements ({requestedValue}).": "",
   "your service's role ({requestedValue}).": "",
   "{0, plural, one {# instance running out of {instancesCount}} other {# instances running out of {instancesCount}}}": "",
+  "{0}": "",
```

Comment:
In case someone is wondering: This is also happening on master, I don't know where it comes from.